kernel/source/core/gpio.c

```diff
@@ -19,67 +19,68 @@
 void InitializeGPIO()
 {
 #ifndef MIOS
-	//set who (ARM or PPC and ARM) can access which GPIO pin
-	//1 = PPC & Starlet
-	//0 = Starlet only
-	set32(HW_GPIO1OWNER, (read32(HW_GPIO1OWNER) & ( 0xFF000000 | GP_PUBLIC )) | GP_OWNER_PPC );
+ //set who (ARM or PPC and ARM) can access which GPIO pin
+ //1 = PPC & Starlet
+ //0 = Starlet only
+	set32(HW_GPIO1OWNER, (read32(HW_GPIO1OWNER) & (0xFF000000 | GP_PUBLIC)) | GP_OWNER_PPC);
 
-	//set output values & directions of the starlet registers
+ //set output values & directions of the starlet registers
```

Comment:
im not sure why these comments are pushed to the start of the line, and are ignoring indentation..

Reply:
fair enough, nothing to be done then :)